### PR TITLE
Reference iOS and .NET apps in README.md

### PR DESCRIPTION
**Pull request by mikewilburn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ An open-source example app demonstrating how to use the ArcGIS platform to colle
 
 Collect your own organization's data in an app driven by the ArcGIS Web GIS information model and configured through your organization's web maps. We provide an example _Trees of Portland_ web map and dataset to get you started.
 
-_Coming summer 2018_
+_Coming summer 2019_
+
+(In the meantime, have a look at our [iOS](https://github.com/Esri/data-collection-ios) and [.NET](https://github.com/Esri/data-collection-dotnet) applications which showcase similar patterns.)
```
Comment (esreli) [inline README.md]: I would suggest not using language that includes "our". Perhaps consider:

(In the meantime, have a look at the corresponding [iOS](https://github.com/Esri/data-collection-ios) and [.NET](https://github.com/Esri/data-collection-dotnet) applications which showcase similar patterns.)

**Review by fallsdownhill — APPROVED** [review]
@mikewilburn works for me

**Review by esreli — COMMENTED** [review]
Perhaps i'm nitpicking, but I think we might want to make one change.

File: README.md (hunk 1 above)
Comment (mikewilburn) [reply README.md]: @esreli not nit-picky at all. I like your version better.

**Review by esreli — APPROVED** [review]
Looks good to me!

**Pull request merged**